test/00_operators/OperatorTests.cu

```diff
@@ -3100,7 +3100,8 @@ TYPED_TEST(OperatorTestsAllExecs, RepMat)
   // Now a rectangular repmat
   tensor_t<TestType, 2> t2r({count0 * same_reps, count1 * same_reps * 2});
 
-  auto rrepop = repmat(t2, {same_reps, same_reps * 2});
+  index_t rreps[] = {same_reps, same_reps * 2};
+  auto rrepop = repmat(t2, static_cast<const index_t*>(rreps));
```

Comment:
Not clear on why this cast is needed. Isn't the type already matched by `repmat`?

Reply:
It says it's ambiguous. Everything else I tried would either match two candidates, or none.

Can I change repmat() to look like slice() permute() etc? Those work fine.

Reply:
It's ambiguous because it looks like there's a pointer overload here: 
```
  /**
   * Repeat a matrix a specific number of times in each direction
   *
   * @tparam T1
   *   Type of operator or view
   * @param t
   *   Operator or view to repeat
   * @param reps
   *   Array of times to repeat in each dimension
   *
   * @returns
   *   New operator with repeated data
   */
  template <typename T1>
    auto __MATX_INLINE__ repmat(T1 t, const index_t (&reps)[])
    {
      return detail::RepMatOp<T1, T1::Rank()>(t, reps);
    };

  /**
   * Repeat a matrix a specific number of times in each direction
   *
   * @tparam T1
   *   Type of operator or view
   * @param t
   *   Operator or view to repeat
   * @param reps
   *   Array of times to repeat in each dimension
   *
   * @returns
   *   New operator with repeated data
   */
  template <typename T1>
    auto __MATX_INLINE__ repmat(T1 t, const index_t *reps)
    {
      return detail::RepMatOp<T1, T1::Rank()>(t, reps);
    };
```

Can you remove the pointer overload function and try it? Not sure why that's there.